src/test/ui/structs/incomplete-fn-in-struct-definition.rs

```diff
@@ -0,0 +1,5 @@
+fn main() {}
+
+struct S {
+    fn //~ ERROR expected identifier, found keyword `fn`
+}
```

Comment:
I think an example like this would be more helpful.
```suggestion
struct S {
    fn: u8, //~ ERROR expected identifier, found keyword `fn`
}
```

Reply:
I fixed it.